### ISSUE TITLE
Today Cards: correct fonts

### ISSUE THREAD

**Issue by deobald** [pariyatti/mobile-app#102]
The original wireframes did have some attractive fonts used in the card samples, but I wasn't sure if those fonts were readily available to mobile systems and if they were the fonts Nishant and @iamsankalp intended to use in the actual application.

I'm creating this card so Nishant and Sankalp can weigh in on which fonts to use. This card is an extension of an issue @brihas raised: #96 

These are these font I chose initially — not because they look nice, obviously. :) Just because it was installed by default on both Android and iOS and they looked _approximately_ like the fonts in the original wireframe ... which is to say, they have serifs.

<img src="https://user-images.githubusercontent.com/20282/179372386-27f0f887-a1f1-416b-a392-6ce61e23a61a.jpeg" width="240px">
<img src="https://user-images.githubusercontent.com/20282/179372389-c146b119-6836-4031-a5fa-cd2835924d1f.jpeg" width="240px">
<img src="https://user-images.githubusercontent.com/20282/179372390-893205d9-c703-4a0f-b316-0ad24cc9675c.jpeg" width="240px">

I will say that the smaller font size on translations probably has to stay, regardless of which font we choose. Any larger, and the translations wrap on almost every single line, which not only looks terrible, it causes some cards to take up the entire screen, hampering usability.

The current font sizes are 20 "logical pixels" for Pāli and 16 "logical pixels" for the translation. If we want the Pāli and its translation to be in identical fonts (and sizes), we could reduce the font size of the Pāli?

**Comment by deobald** [comment]
I've removed the bold as per #96 ... the font sizes still differ. For example:

<img src="https://user-images.githubusercontent.com/20282/179372740-787f04b2-0915-46a1-8d24-a1ea6bf05ff6.jpeg" width="240px">

**Comment by deobald** [comment]
Feedback from Sankalp:

From what I could check, the fonts used are:

* Sans Serif: Apple system font i.e. SF Pro
* Serif: Georgia

So, this would need to be tweaked for Android anyway since SF Pro won't be available.

**Comment by deobald** [comment]
Here are tentative choices:

* San Serif: [Inter](https://rsms.me/inter/) / [Inter on Google Fonts](https://fonts.google.com/specimen/Inter)
* Serif: in lieu of Georgia (which is non-free) either [Merriweather](https://fonts.google.com/specimen/Merriweather?category=Serif) or [Gentium](https://software.sil.org/gentium/design/)

I'll create screenshots with each and show them to Sankalp for review.

**Comment by deobald** [comment]
Note that `Gentium Plus` is not currently available through the `google_fonts` Flutter package. (Though it should be soon.)

Here are the Serif font samples. Caveats:

* Note that the "px" in each is Flutter's "logical pixels". 
* Georgia is not necessarily going to provide the real Microsoft Georgia font. Since Georgia is non-free, it isn't available from the `google_fonts` Flutter package. `fontFamily: "Georgia"` is permitted, but I think it's safe to say the results are non-deterministic. For that reason, I'd suggest we look at the other two fonts.
* Gentium rendered quite small, so I also took a screenshot at 18px.

## Georgia Font Family 16px:
<img src="https://user-images.githubusercontent.com/20282/182741043-6a98d633-b613-40ee-a87f-897f4dfdb3ff.jpeg" width="340"/>

## Gentium Book Basic 16px:
<img src="https://user-images.githubusercontent.com/20282/182741037-61e573e8-cb6f-45cc-9605-6e2f9d859bd8.jpeg" width="340"/>

## Gentium Book Basic 18px:
<img src="https://user-images.githubusercontent.com/20282/182741041-64e50ea3-eeee-4abb-b481-6cdf428ca952.jpeg" width="340"/>

## Merriweather 16px:
<img src="https://user-images.githubusercontent.com/20282/182741045-a97e1a0f-5edc-4dbc-abda-0c2b233ac44b.jpeg" width="340"/>

I've reduced these to 340px wide to help provide _some_ sense of font scaling at lower resolutions. However, that's quite artificial — the above images are what fonts look like scaled by the browser, not a mobile operating system. The reduced resolution is only representative of font weight over pixel territory. For example, Merriweather is "fatter" and will use more pixels regardless of whether iOS or Firefox is scaling it.

To see the full-sized screenshots taken on an iPhone SE (2020), use the following links. I found it's easiest to download these and compare side-by-side.

[georgia-family-16px](https://user-images.githubusercontent.com/20282/182741043-6a98d633-b613-40ee-a87f-897f4dfdb3ff.jpeg)
[gentium-book-basic-16px](https://user-images.githubusercontent.com/20282/182741037-61e573e8-cb6f-45cc-9605-6e2f9d859bd8.jpeg)
[gentium-book-basic-18px](https://user-images.githubusercontent.com/20282/182741041-64e50ea3-eeee-4abb-b481-6cdf428ca952.jpeg)
[merriweather-16px](https://user-images.githubusercontent.com/20282/182741045-a97e1a0f-5edc-4dbc-abda-0c2b233ac44b.jpeg)

**Comment by deobald** [comment]
Here are the Sans Serif font samples.

These aren't strictly necessary, since @iamsankalp already decided on `Inter` ... however, I'm including them for posterity and for the interest of anyone reading this issue. I was quite surprised to find out how much nicer `Inter` looks than `Noto Sans`.

## Noto Sans:

<img src="https://user-images.githubusercontent.com/20282/182743753-30248f73-ffd6-4db1-846c-4b9d5ae080f1.jpeg" width="340" />

<img src="https://user-images.githubusercontent.com/20282/182743750-2af66ce2-864f-444f-a816-5b7869c55d90.jpeg" width="340" />

Note that the headers (the date and _"PALI WORD OF THE DAY"_) are still in the default system font. It's definitely much easier to see why `Inter` is superior when looking at the full-res versions side-by-side:

[pali-noto-sans-18px](https://user-images.githubusercontent.com/20282/182743753-30248f73-ffd6-4db1-846c-4b9d5ae080f1.jpeg)
[pali-inter-18px](https://user-images.githubusercontent.com/20282/182743750-2af66ce2-864f-444f-a816-5b7869c55d90.jpeg)

**Comment by deobald** [comment]
The _Settings_ tabs remain a slightly weird font because there doesn't seem to be a way to set the font on the `Tab` class. @iamsankalp is designing a new _Account_ screen anyway, which will likely do away with the tabs.

**Comment by deobald** [comment]
All fonts now default to `Inter` across the app. `Gentium Book Basic 18 lpx` is the font I'll leave in place for serif fonts until @iamsankalp evaluates them.

**Comment by deobald** [comment]
Closing this issue as the current fonts are working quite well. If @iamsankalp wants, we can open another issue to tweak the font choices.

**Comment by brihas** [comment]
Merriweather is not an option as it does not contain all the characters
needed to display Pāli words, so I suspect it is not fully Unicode
compliant. For example, please refer to the ṃ (m underdot) in your
screenshot and you will see that it is being replaced by a sans-serif font
since ṃ cannot be displayed with Merriweather.

For testing purposes, it would be good to verify that any font under
consideration can in fact display all the characters we require for
displaying Pāli. Only once it passes this check should it be considered.
